core/src/main/java/com/linecorp/armeria/server/docs/ServiceInfo.java

```diff
@@ -74,11 +80,7 @@ public ServiceInfo(String name,
                        @Nullable String docString) {
 
         this.name = requireNonNull(name, "name");
-
-        requireNonNull(methods, "methods");
-
-        this.methods = ImmutableSortedSet.copyOf(comparing(MethodInfo::name)
-                                                         .thenComparing(MethodInfo::httpMethod), methods);
+        this.methods = ImmutableList.copyOf(mergeEndpoints(requireNonNull(methods)));
```

Comment:
So there's no reason to use list here?

Reply:
you're right. reverted back to an `ImmutableSortedSet`!